css/styles.css

```diff
@@ -0,0 +1,938 @@
+button::-moz-focus-inner,[type="button"]::-moz-focus-inner,[type="reset"]::-moz-focus-inner,[type="submit"]::-moz-focus-inner{border-style:none;padding:0}
+button:-moz-focusring,[type="button"]:-moz-focusring,[type="reset"]:-moz-focusring,[type="submit"]:-moz-focusring{outline:1px dotted ButtonText}
+fieldset{border:1px solid #c0c0c0;margin:0 2px;padding:0.35em 0.625em 0.75em}legend{box-sizing:border-box;color:inherit;display:table;max-width:100%;padding:0;white-space:normal}
+progress{display:inline-block;vertical-align:baseline}textarea{overflow:auto}[type="checkbox"],[type="radio"]{box-sizing:border-box;padding:0}
+[type="number"]::-webkit-inner-spin-button,[type="number"]::-webkit-outer-spin-button{height:auto}[type="search"]{-webkit-appearance:textfield;outline-offset:-2px}
+[type="search"]::-webkit-search-cancel-button,[type="search"]::-webkit-search-decoration{-webkit-appearance:none}
+::-webkit-file-upload-button{-webkit-appearance:button;font:inherit}details,menu{display:block}summary{display:list-item}
+canvas{display:inline-block}template{display:none}[hidden]{display:none}
+
+
+/*-------------------GENERAL */
+
+body,html{
+	background:#E8DBD2;
+	overflow-x: hidden;
+}
+
+body, a, a:hover {
+	cursor: url("../images/cursor.png"), progress !important;
+}
+
+@font-face {
+	font-family: Changa One;
+	src: url('../fonts/changa_one/ChangaOne-Regular.ttf');
+}
+
+@font-face {
+	font-family: Vesper Libre;
+	src: url('../fonts/vesper_libre/VesperLibre-Regular.ttf');
+}
+
+@font-face {
+	font-family: Pragati Narrow;
+	src: url('../fonts/pragati_narrow/PragatiNarrow-Regular.ttf');
+}
+
+h1{
+	font-family: Changa One, sans-serif;
+	color: #F05559;
+	font-size: 30pt;
+	text-transform: uppercase;
+	font-weight: lighter;
+	letter-spacing: 17px;
+	margin: 0px;
+}
+
+h2 {
+	font-family: Pragati Narrow, serif;
+	color: #1C2EA3;
+	font-size: 13.6pt;
+	font-weight: lighter;
+	letter-spacing: -0.1px;
+	line-height: 25px;
+}
+
+h3 {
+	font-family: Changa One, sans-serif;
+	color: #000;
+	font-size: 14pt;
+	font-weight: lighter;
+}
+
+p {
+	font-family: Vesper Libre, serif;
+	color: #000;
+	font-size: 10pt;
+	font-weight: lighter;
+	line-height: 22px;
+}
+
+
+
+
+/*-------------------------------*/
+/*----------------------HOMEPAGE */
+/*-------------------------------*/
+
+/*---------------------MENU */
+
+section{
+	margin: 30px;
+}
+
+#menu{
+	display: flex;
+	justify-content:center;
+}
+
+#menu > div a:link{
+	margin-left: 40px;
+	margin-right: 40px;
+	font-family: Changa One, sans-serif;
+	color: #E8DBD2;
+	font-size: 21pt;
+	text-transform: uppercase;
+	text-decoration: none;
+	letter-spacing: 2px;
+	/* line text */
+  	text-shadow:
+   	-1px -1px 0 #F05559,  
+    1px -1px 0 #F05559,
+    -1px 1px 0 #F05559,
+    1px 1px 0 #F05559;
+}
+
+#menu > div a:visited{
+	color: #E8DBD2;
+	text-decoration: none;
+	text-transform: none;
+}
+
+#menu > div a:active{
+	color: #E8DBD2;
+	text-decoration: none;
```

Comment:
this is causing the menu items to revert to lower-case when clicked, these should probably remain uppercase the entire time no?